README.md

```diff
@@ -154,6 +154,47 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Deploying a Custom Image to an OpenShift Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to a
+ OpenShift Cluster for others to review.  This involves building a local image, pushing the
+ image to an image registry, then updating the OpenShift cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. An image registry like [quay.io ](https://quay.io/signin/) or [dockerhub](https://hub.docker.com/)
+
+##### Steps
+1. Create a repository in the image registry of your choice to hold the image
+  ```
+  Login to https://quay.io, and create a a empty public repository 'console'
+  ```
+2. Build Image `docker build -t <your-image-name> [<your-git-url> | <path-to-dockerfile>]`
+  ```
+  docker build -t quay.io/myaccount/console github.com/myaccount/console#latestchanges
```

Comment:
Better to explicitly include a tag in these examples like

```suggestion
  docker build -t quay.io/myaccount/console:latest github.com/myaccount/console#latestchanges
```

(Same applies to the commands below.)